docs/getting_started-new/index.md

```diff
@@ -1,7 +1,180 @@
+scenario's execution graph made of two data nodes (blue boxes) and one task 
+(orange box):
+
+![hello world example](hello_world.svg){width=50%}
+
+It consists in one input data node named *name*. Then a task named 
+*build_message* takes the first data node and returns a second 
+data node named *message*.
+
+Building the corresponding Taipy application requires three easy steps.
+
+### Configuring the application
+
+The first step involves configuring the execution graph. This includes 
+configuring data nodes, tasks, and scenario itself. 
+Use the following Python code:
+
+```python linenums="1"
+from taipy import Config
+
+
+def build_message(name: str):
+    return f"Hello {name}!"
+
+
+name_data_node_cfg = Config.configure_data_node(id="input_name")
+message_data_node_cfg = Config.configure_data_node(id="message")
+build_msg_task_cfg = Config.configure_task("build_msg", build_message, name_data_node_cfg, message_data_node_cfg)
+scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_task_cfg])
+```
+
+- Lines 4-5 define the function that the task will use during execution.
+- Lines 8-9 configure the two data nodes, *input_name* and *message*.
+- Line 10 configures a task named *build_msg* associated with the `build_message()`
+  function, specifying input and output data nodes.
+- Finally, line 11 configures the execution graph of the scenario providing 
+  the previously configured task.
+
+### Running The Core service
+
+Running the Core service allows Taipy to process the configuration made in the 
+previous step and set up the scenario management feature.
+
+Include the following code in your Python script:
+
+``` python linenums="1"
+from taipy import Core
+
+if __name__ == "__main__":
+    Core().run()
+```
+Line 3 is standard boilerplate code to ensure that your script runs only from the 
+main module, preventing accidental execution. It is strongly recommended to include it.
+
+In line 4, we simply instantiate and run a Core service.
+
+### Creating Scenarios and accessing data
+
+Now that Taipy Core service is running, you can create and manage scenarios, submit 
+task graphs for execution, and access data nodes. Here's a code snippet to demonstrate 
+these actions:
+
+
+```python linenums="1"
+import taipy as tp
+
+hello_scenario = tp.create_scenario(scenario_cfg)
+hello_scenario.input_name.write("Taipy")
+hello_scenario.submit()
+print(hello_scenario.message.read())
+```
+
+In line 3, method `tp.create_scenario()` instantiates the new scenario name 
+`hello_scenario` from the scenario configuration built before.
+
+Line 4, sets the input data node *input_name* of `hello_scenario` with the string 
+value "Taipy" using the write() method.
+
+Line 5 submits the `hello_scenario` for execution, which triggers the creation 
+and execution of a job. This job reads the input data node, passes the value 
+to the `build_message()` function, and writes the result to the output data node.
```

Comment:
same
... then in the rest of the text body.

Only API elements should have a fixed size font.